docs/configuration/upstream/propose_downstream.md

```diff
@@ -32,7 +40,87 @@ jobs:
 - job: propose_downstream
   trigger: release
   dist_git_branches:
-    - f35
+    - f39
+```
+
+This config would update Fedora Rawhide and Fedora 39 dist-git branches.
+
+
+
+## Syncing the release to CentOS Stream
+
+Recently, we have implemented support for `propose-downstrem` also for CentOS Stream. This means that you can now configure
+the upstream release to trigger release syncing the same way as for Fedora to CentOS Stream dist-git repositories
+hosted in https://gitlab.com/redhat/centos-stream/rpms.
+
+To sync the release to CentOS Stream, you currently need to specify the following parameters:
+- `pkg_tool: centpkg`  
+- `dist_git_branches`:  `c8s` and/or `c9s`. 
+
+Example of such a configuration:
+```yaml
+---
+downstream_package_name: nispor
+upstream_project_url: https://github.com/nispor/nispor
+issue_repository: https://github.com/nispor/nispor
+update_release: false
+upstream_package_name: nispor
+upstream_tag_template: v{version}
+copy_upstream_release_description: false
+
+
+pkg_tool: centpkg
+actions:
+  post-upstream-clone:
+    - wget https://gitlab.com/redhat/centos-stream/rpms/nispor/-/raw/c9s/nispor.spec
+
+jobs:
+  - job: propose_downstream
+    trigger: release
+    dist_git_branches:
+      - c9s
```

Comment:
or do you think this is confusing and we should just directly advise having the 2 packages below?

Reply:
I don't really mind it, the problem is that we cannot _overload_ the config in the Service, so this looks nice, but you can only use it locally and with explicitly specified config (in case you have Fedora set up too) :/

Maybe with a disclaimer that this can be used as a minimal config with CLI with `--config` switch

Reply:
hmm, in that case, I would rather remove it to not create unnecessary confusion